crypto/ec/ec_asn1.c

```diff
@@ -887,6 +887,9 @@ EC_GROUP *d2i_ECPKParameters(EC_GROUP **a, const unsigned char **in, long len)
         return NULL;
     }
 
+    if (params->type == 1)
```

Comment:
Now would be a good time to clean up this preexisting magic number rubbish :)

Reply:
Makes sense. Done